### PR TITLE
Update getting started

### PR DESCRIPTION
**Pull request by monicasarbu**
- Add in the overview that it's installed as an agent
- Make sure Elasticsearch, Kibana and optionally Logstash are installed before

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -65,7 +27,6 @@ filebeat:
   prospectors: 1024
 -------------------------------------------------------------------------------------
 
-==== Prospector
 Every prospector item stats with a -. Inside are the prospector specific configs defined
```
Comment (andrewkroh) [inline docs/configuration.asciidoc]: s/stats/starts/

File: docs/getting-started.asciidoc [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-
- * Install Filebeat on your servers to make use of your log files. <<filebeat-installation>>
- * Install Elasticsearch for storage and indexing the data. {libbeat}/getting-started.html#elasticsearch-installation[More details]
- * Install Kibana for the UI. {libbeat}/getting-started.html#kibana-installation[More details]
- * Install optionally Logstash. {libbeat}/getting-started.html#logstash-installation[More details]
+Get started with your own Filebeat setup by first:
 
+ * installing Elasticsearch for storage and indexing the data. {libbeat}/getting-started.html#elasticsearch-installation[More details]
+ * installing Kibana for the UI. {libbeat}/getting-started.html#kibana-installation[More details]
+ * installing optionally Logstash. {libbeat}/getting-started.html#logstash-installation[More details]
```
Comment (ruflin) [inline docs/getting-started.asciidoc]: optional but recommended for filebeat

**Comment by ruflin** [comment]
LGTM

File: docs/getting-started.asciidoc [hunk 3]
```diff
@@ -74,6 +72,51 @@ filebeat:
...
+    -
+      paths:
+        - /var/log/system.log
+        - /var/log/wifi.log
+    -
+      paths:
+        - /var/log/apache/*
+-------------------------------------------------------------------------------------
+
+The config file above will start to prospectors. The first prospector will have two harvesters,
```
Comment (andrewkroh) [inline docs/getting-started.asciidoc]: `s/to/two/`

File: docs/overview.asciidoc [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 
 Filebeat is a log data shipper initially based on the https://github.com/elastic/logstash-forwarder[Logstash-Forwarder]
-source code. It monitors log directories or specific log files, tails them and forwards them to
+source code. Installed as an agent on your servers, Filebeat monitors the log directories or specific log files, tails them and forwards them to
```
Comment (andrewkroh) [inline docs/overview.asciidoc]: There was some previous discussion around agent vs shipper. This paragraph uses both terms. Just wanted to draw your attention to this.

**Comment by andrewkroh** [comment]
LGTM. Just left a few minor comments.

**Comment by monicasarbu** [comment]
Ready to merge

**Pull request merged**